packages/framework/src/Controller/Admin/AdministratorController.php

```diff
@@ -79,12 +79,21 @@ public function __construct(
     #[CanView]
     public function listAction(): Response
     {
-        $queryBuilder = $this->administratorFacade->getAllListableQueryBuilder();
+        if ($this->getCurrentAdministrator()->isSuperadmin()) {
+            $queryBuilder = $this->administratorFacade->getAllQueryBuilder();
+        } else {
+            $queryBuilder = $this->administratorFacade->getAllListableExcludingSuperadminQueryBuilder();
+        }
         $dataSource = $this->queryBuilderDataSourceFactory->create($queryBuilder, 'a.id');
 
         $grid = $this->gridFactory->create('administratorList', $dataSource, AdminRoleConstant::ROLE_ADMINISTRATOR);
         $grid->setDefaultOrder('realName');
 
+        if ($this->getCurrentAdministrator()->isSuperadmin()) {
```

Comment:
I would move this column to the end after the email column. It looks weird to have this information as the first column. The first should be a row identifier, and after that, any other information associated with it.

Reply:
🤷‍♂️  changed ea92bba420